source/draw/RasterUv.ooc

```diff
@@ -44,9 +44,14 @@ RasterUv: class extends RasterPacked {
 			uv referenceCount decrease()
 	}
 	fill: override func (color: ColorRgba) {
-		for (y in 0 .. this size y)
-			for (x in 0 .. this size x)
-				this[x, y] = ColorUv new(color r, color g)
+		sizeY := this size y
+		sizeX := this size x
+		buffer := this buffer pointer as ColorUv*
+		stride := this stride / this bytesPerPixel
```

Comment:
This isn't the same as `this stride` obviously, but it should still have some other name.